test-packages/self-tests/tsd-test/check-if-tsd-is-working.ts

```diff
@@ -0,0 +1,24 @@
+import { execSync } from 'child_process';
+import { resolve } from 'path';
+import { createLogger } from '@sap-cloud-sdk/util';
+
+const logger = createLogger('check-if-dtslint-are-working');
+try {
```

Comment:
[pp]I would expect this to work without us explicitly checking it so perhaps it is redundant.